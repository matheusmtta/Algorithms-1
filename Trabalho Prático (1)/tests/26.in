20 i
42 i
16 i
44 i
44 r
71 i
54 i
71 r
218 p
16 r
68 p
42 r
66 p
250 p
26 p
18 i
118 p
120 p
8 p
20 r
288 p
30 i
22 i
66 i
166 p
54 r
66 r
78 p
22 r
7 i
107 p
30 r
15 i
15 r
68 i
329 p
68 r
38 i
7 r
45 i
5 i
39 i
24 i
41 i
67 p
1 i
39 r
63 i
34 p
24 r
60 p
34 i
60 i
46 i
17 i
69 i
51 i
38 r
229 p
37 i
13 i
226 p
45 r
46 r
29 i
63 r
13 r
1 r
17 r
29 r
134 p
