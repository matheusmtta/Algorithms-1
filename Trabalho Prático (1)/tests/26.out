6
4
7
6
5
6
6
6
6
5
3
6
7
3
3
3
4
4
3
